diff --git a/net/miniupnpd-iptables/files/miniupnpd.init b/net/miniupnpd-iptables/files/miniupnpd.init
index c5a14ab..5a0696f 100644
--- a/net/miniupnpd-iptables/files/miniupnpd.init
+++ b/net/miniupnpd-iptables/files/miniupnpd.init
@@ -68,7 +68,8 @@ upnpd() {
 	local extiface intiface upload download logging secure enabled natpmp
 	local extip port usesysuptime conffile serial_number model_number
 	local uuid notify_interval presentation_url enable_upnp
 	local upnp_lease_file clean_ruleset_threshold clean_ruleset_interval
 	local args ifname ipv6_listening_ip enabled
+	local force_forwarding
 
 	config_get_bool enabled config enabled 1
@@ -93,5 +93,6 @@ upnpd() {
 	config_get clean_ruleset_threshold config clean_ruleset_threshold
 	config_get clean_ruleset_interval config clean_ruleset_interval
 	config_get ipv6_listening_ip config ipv6_listening_ip
+	config_get_bool force_forwarding config force_forwarding 0
 
 	. /lib/functions/network.sh
@@ -138,5 +138,6 @@ upnpd() {
 		upnpd_write_bool secure_mode 1
 		upnpd_write_bool pcp_allow_thirdparty 0
 		upnpd_write_bool system_uptime 1
+		upnpd_write_bool force_forwarding 0 ext_allow_private_ipv4
 
 		[ -n "$upnp_lease_file" ] && \
diff --git a/net/miniupnpd-iptables/files/upnpd.config b/net/miniupnpd-iptables/files/upnpd.config
index bd7c3ec..b03d60a 100644
--- a/net/miniupnpd-iptables/files/upnpd.config
+++ b/net/miniupnpd-iptables/files/upnpd.config
@@ -2,6 +2,7 @@ config upnpd config
 	option enabled		0
 	option enable_natpmp	1
 	option enable_upnp	1
+	option force_forwarding	0
 	option secure_mode	1
 	option log_output	0
 	option download		1024
